### ISSUE TITLE
Bulk deleting pages does not remove aliases

### ISSUE THREAD

**Issue by smellydelli** [strapi-community/strapi-plugin-url-alias#3]
### Steps to reproduce the behavior

1. Go to "Content Manager"
2. Tick the checkbox at the top left of the page list
3. Click the "Delete" button
4. Go to settings -> URL alias -> list
5. Aliases remain

This only happens if you use the "delete" button at the top. The bin icon on the right hand side of the listing works ok, as well as deleting from within the page itself.

### Expected behavior

Expected behaviour is URL aliases should be removed when pages are deleted.

### Screenshots

![Screenshot-2022-07-09-at-16 12 37](https://user-images.githubusercontent.com/12031410/178111928-66accb5f-1196-49a5-8e4c-ea5a5a98c0f1.jpg)

### System

- Node.js version: 16.16.0 LTS
- NPM version: 8.11.0
- Strapi version: 4.2.2
- Plugin version: 1.0.0-alpha.2
- Database: sqlite
- Operating system: Alpine Linux

**Comment by boazpoolman** [comment]
I'm working on this RN. I'll probably have a fix within a couple of days.

**Comment by boazpoolman** [comment]
Hi @smellydelli,

I have fixed this in PR #11.
Could you test it out locally?
You can install the PR in your project like so:

```
yarn add strapi-community/strapi-plugin-url-alias#pull/11/head
npm install strapi-community/strapi-plugin-url-alias#pull/11/head
```

**Comment by boazpoolman** [comment]
This was released with version `1.0.0-alpha.5` of the plugin.
If you're still experiencing the issue beyond this version feel free to re-open the GH issue or create a new one.
